### DISCUSSION TITLE
Using google SMTP to send email

### DISCUSSION THREAD

**Discussion by UnconnectedBedna** [dani-garcia/vaultwarden#4262] [Q&A] [answered]
The desctiption on the readme references the old method setting up to use gmail smtp servers to send mail.

I figured out you can still do it another way by following this tutorial:
https://www.androidpolice.com/how-to-allow-less-secure-apps-in-gmail/

I took a quick look here in the discussions form and could not see anywone providing a solution since google removed the "less secure apps" in 2022.

Hope someone else finds this useful. :)

```
host = smtp.gmail.com
Secure SMTP = force_tls
Port = 465
Username = YOUR_GOOGLE_LOGIN
Password = APP_PASSWORD_YOU_GENERATED
```

**Comment by stefan0xC** [comment, marked as answer]
> The desctiption on the readme references the old method setting up to use gmail smtp servers to send mail.

What [readme](https://github.com/dani-garcia/vaultwarden/blob/main/README.md) are you referring to? The [SMTP-Configuration](https://github.com/dani-garcia/vaultwarden/wiki/SMTP-Configuration#googlegmail)? Is [the referenced link](https://support.google.com/accounts/answer/185833?hl=en&ref_topic=7189145) not correct? Last time I tried it, it still worked as explained.

**Comment by UnconnectedBedna** [comment]
I am referencing the wiki you linked to.
And the "referenced link" you included in your answer is awsome, if it actually was included in the wiki page. xD

Only links on the wiki are:
https://support.google.com/accounts/answer/185833?hl=en&ref_topic=7189145
and
https://web.archive.org/web/20210925161633/https://webewizard.com/2019/09/17/Using-Lettre-With-Gmail/

**edit**
wtf, now the top link is ALSO taking me to the same page you linked, when I pressed it earlier it brought me to a page explaining "less secure apps" are discontinued.
OR, more likely, I was an idiot and clicked the wrong tab after opening it.

Never the less, you are correct, and this whole thread can be removed.

**Reply by stefan0xC** [reply]
Weird. I mean someone also commented that it didn't work for them. 🤔

**Reply by UnconnectedBedna** [reply]
To be clear, the "send test email" works, so I just assume invite mails will also work.
It is mentioned in the tutorial I posted that this method will also be removed further down the line. I should probably look into using another provider.
